mkdocs.yml

```diff
@@ -66,13 +66,22 @@ pages:
             - 'REST API reference': 'api/rest_api_reference.md'
         - Public PHP API:
             - 'Public PHP API': 'api/public_php_api.md'
-        - 'Field Type API and best practices': 'api/field_type_api_and_best_practices.md'
+        - Field Type:
+            - 'Reference': 'api/field_type_reference.md'
+            - 'Field Type API': 'api/field_type_api_and_best_practices.md'
```

Comment:
Actually, maybe get rid of "best practices" here and in the page itself? I don't think we have any.

Reply:
Also, maybe put this page in first place under "Field Type", even before "Reference"?

Reply:
done